.github/workflows/data-checks.yml

```diff
@@ -22,9 +22,11 @@ jobs:
       # one execution of the tests per version listed above
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Upgrade wheel
```

Comment:
Seems like wheel is an "[optimization](https://pip.pypa.io/en/stable/cli/pip_wheel/)"

Seems safe to remove

Reply:
This is not obvious if you're not a pip person, but `install -U` upgrades, not removes. Since basically everything depends on wheel somewhere now, I'm just making sure we have the most current one.